### PR TITLE
🌐 Add Korean translation for Tutorial - First Steps

### PR DESCRIPTION
**Pull request by hard-coders**
This PR translates `tutorial/first-steps.md` in Korean.

related: #2017

### CODE REVIEW

File: docs/ko/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+### 대화형 API 문서
+
+이제 <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>로 가봅니다.
+
+자동 대화형 API 문서를 볼 수 있습니다 (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a> 제공):
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+
+### 대안 API 문서
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/first-steps.md:60]:
```suggestion
### 대체 API 문서
```

File: docs/ko/docs/tutorial/first-steps.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+그리고 이제, <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>로 가봅니다.
+
+대안 자동 문서를 볼 수 있습니다 (<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a> 제공):
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/first-steps.md:64]:
```suggestion
대체 자동 문서를 볼 수 있습니다 (<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a> 제공):
```

File: docs/ko/docs/tutorial/first-steps.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+이러한 경우 JSON 속성, 가지고 있는 데이터 타입 등을 뜻합니다.
+
+#### OpenAPI와 JSON 스키마
+
+OpenAPI는 API에 대한 API 스키마를 정의합니다. 또한 이 스키마에는 JSON 데이터 스키마의 표준인 **JSON 스키마**를 사용하여 API에서 보내고 받은 데이터의 정의(또는 "스키마")를 포함합니다.
+
+#### `openapi.json` 확인
+
+날(raw) OpenAPI 스키마가 어떻게 생겼는지 궁금하다면, FastAPI는 자동으로 API의 설명과 함께 JSON (스키마)를 생성합니다.
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/first-steps.md:94]:
```suggestion
가공되지 않은 OpenAPI 스키마가 어떻게 생겼는지 궁금하다면, FastAPI는 자동으로 API의 설명과 함께 JSON (스키마)를 생성합니다.
```

File: docs/ko/docs/tutorial/first-steps.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+...
+```
+
+#### OpenAPI의 용도
+
+OpenAPI 스키마는 포함된 두 개의 대화형 문서 시스템을 제공합니다.
+
+그리고 OpenAPI의 모든 것을 기반으로 하는 수십 가지 대안이 있습니다. **FastAPI**로 빌드한 애플리케이션에 이러한 대안을 쉽게 추가 할 수 있습니다.
+
+API와 통신하는 클라이언트를 위해 코드를 자동으로 생성하는 데도 사용할 수 있습니다. 예로 프런트 앤드, 모바일, IoT 애플리케이션이 있습니다.
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/first-steps.md:127]:
```suggestion
API와 통신하는 클라이언트를 위해 코드를 자동으로 생성하는 데도 사용할 수 있습니다. 예로 프론트엔드, 모바일, IoT 애플리케이션이 있습니다.
```

File: docs/ko/docs/tutorial/first-steps.md [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+...경로는 다음과 같습니다:
+
+```
+/items/foo
+```
+
+!!! info "정보"
+    "경로"는 일반적으로 "앤드포인트" 또는 "라우트"라고도 불립니다.
+
+API를 빌드하는 동안 "경로"는 "우려사항"과 "리소스"를 분리하는 주요 방법입니다.
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/first-steps.md:205]:
```suggestion
API를 빌드하는 동안 "경로"는 "관심사"와 "리소스"를 분리하는 주요 방법입니다.
```

File: docs/ko/docs/tutorial/first-steps.md [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+
+---
+
+`async def` 대신 일반 함수로 정의할 수 있습니다:
+
+```Python hl_lines="7"
+{!../../../docs_src/first_steps/tutorial003.py!}
+```
+
+!!! 참고
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/first-steps.md:312]:
```suggestion
!!! note 참고
```

**Review by PandaHun — COMMENTED** [review]
It's almost there!

File: docs/ko/docs/tutorial/first-steps.md (hunk 1 above)
Comment (hard-coders) [reply docs/ko/docs/tutorial/first-steps.md:60]: "대체" is more closer to "replace", not "alternative".

File: docs/ko/docs/tutorial/first-steps.md (hunk 5 above)
Comment (hard-coders) [reply docs/ko/docs/tutorial/first-steps.md:205]: Oops...

**Review by PandaHun — APPROVED** [review]

**Review by Suyoung789 — APPROVED** [review]

**Comment by tiangolo** [comment]
Great! Thank you @hard-coders ! :bow: :rocket: 

And thanks for the reviews @PandaHun and @Suyoung789 ! :cake:

**Pull request merged**